### ISSUE TITLE
Could not pass event InventoryOpenEvent to AutomaticInventory v2.8.5

### ISSUE THREAD

**Issue by clinot** [MLG-Fortress/AutomaticInventory#14]
Automatic Inventory is v2.8.5 and Spigot is API version 1.14.1 R0.1 Snapshot

```
21.05 10:24:32 [Server] Server thread/ERROR Could not pass event InventoryOpenEvent to AutomaticInventory v2.8.5
21.05 10:24:32 [Server] INFO org.bukkit.event.EventException: null
21.05 10:24:32 [Server] INFO at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:320) ~[spigot-1.14.1.jar:git-Spigot-8043ebc-f690957]
21.05 10:24:32 [Server] INFO at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[spigot-1.14.1.jar:git-Spigot-8043ebc-f690957]
21.05 10:24:32 [Server] INFO at org.bukkit.plugin.SimplePluginManager.fireEvent(SimplePluginManager.java:520) ~[spigot-1.14.1.jar:git-Spigot-8043ebc-f690957]
21.05 10:24:32 [Server] INFO at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:505) ~[spigot-1.14.1.jar:git-Spigot-8043ebc-f690957]
21.05 10:24:32 [Server] INFO at org.bukkit.craftbukkit.v1_14_R1.event.CraftEventFactory.callInventoryOpenEvent(CraftEventFactory.java:1059) ~[spigot-1.14.1.jar:git-Spigot-8043ebc-f690957]
21.05 10:24:32 [Server] INFO at net.minecraft.server.v1_14_R1.EntityPlayer.openContainer(EntityPlayer.java:1042) ~[spigot-1.14.1.jar:git-Spigot-8043ebc-f690957]
21.05 10:24:32 [Server] INFO at net.minecraft.server.v1_14_R1.BlockChest.interact(BlockChest.java:202) ~[spigot-1.14.1.jar:git-Spigot-8043ebc-f690957]
21.05 10:24:32 [Server] INFO at net.minecraft.server.v1_14_R1.IBlockData.interact(SourceFile:282) ~[spigot-1.14.1.jar:git-Spigot-8043ebc-f690957]
21.05 10:24:32 [Server] INFO at net.minecraft.server.v1_14_R1.PlayerInteractManager.a(PlayerInteractManager.java:463) ~[spigot-1.14.1.jar:git-Spigot-8043ebc-f690957]
21.05 10:24:32 [Server] INFO at net.minecraft.server.v1_14_R1.PlayerConnection.a(PlayerConnection.java:1263) ~[spigot-1.14.1.jar:git-Spigot-8043ebc-f690957]
21.05 10:24:32 [Server] INFO at net.minecraft.server.v1_14_R1.PacketPlayInUseItem.a(PacketPlayInUseItem.java:27) ~[spigot-1.14.1.jar:git-Spigot-8043ebc-f690957]
21.05 10:24:32 [Server] INFO at net.minecraft.server.v1_14_R1.PacketPlayInUseItem.a(PacketPlayInUseItem.java:1) ~[spigot-1.14.1.jar:git-Spigot-8043ebc-f690957]
21.05 10:24:32 [Server] INFO at net.minecraft.server.v1_14_R1.PlayerConnectionUtils.lambda$0(PlayerConnectionUtils.java:13) ~[spigot-1.14.1.jar:git-Spigot-8043ebc-f690957]
21.05 10:24:32 [Server] INFO at net.minecraft.server.v1_14_R1.TickTask.run(SourceFile:18) [spigot-1.14.1.jar:git-Spigot-8043ebc-f690957]
21.05 10:24:32 [Server] INFO at net.minecraft.server.v1_14_R1.IAsyncTaskHandler.executeTask(SourceFile:135) [spigot-1.14.1.jar:git-Spigot-8043ebc-f690957]
21.05 10:24:32 [Server] INFO at net.minecraft.server.v1_14_R1.IAsyncTaskHandlerReentrant.executeTask(SourceFile:23) [spigot-1.14.1.jar:git-Spigot-8043ebc-f690957]
21.05 10:24:32 [Server] INFO at net.minecraft.server.v1_14_R1.IAsyncTaskHandler.executeNext(SourceFile:114) [spigot-1.14.1.jar:git-Spigot-8043ebc-f690957]
21.05 10:24:32 [Server] INFO at net.minecraft.server.v1_14_R1.MinecraftServer.aW(MinecraftServer.java:895) [spigot-1.14.1.jar:git-Spigot-8043ebc-f690957]
21.05 10:24:32 [Server] INFO at net.minecraft.server.v1_14_R1.MinecraftServer.executeNext(MinecraftServer.java:888) [spigot-1.14.1.jar:git-Spigot-8043ebc-f690957]
21.05 10:24:32 [Server] INFO at net.minecraft.server.v1_14_R1.IAsyncTaskHandler.awaitTasks(SourceFile:123) [spigot-1.14.1.jar:git-Spigot-8043ebc-f690957]
21.05 10:24:32 [Server] INFO at net.minecraft.server.v1_14_R1.MinecraftServer.sleepForTick(MinecraftServer.java:872) [spigot-1.14.1.jar:git-Spigot-8043ebc-f690957]
21.05 10:24:32 [Server] INFO at net.minecraft.server.v1_14_R1.MinecraftServer.run(MinecraftServer.java:805) [spigot-1.14.1.jar:git-Spigot-8043ebc-f690957]
21.05 10:24:32 [Server] INFO at java.lang.Thread.run(Thread.java:748) [?:1.8.0_191]
21.05 10:24:32 [Server] INFO Caused by: java.lang.NoSuchMethodError: org.bukkit.inventory.Inventory.getName()Ljava/lang/String;
21.05 10:24:32 [Server] INFO at me.ryanhamshire.AutomaticInventory.AIEventHandler.isSortableChestInventory(AIEventHandler.java:392) ~[?:?]
21.05 10:24:32 [Server] INFO at me.ryanhamshire.AutomaticInventory.AIEventHandler.onInventoryOpen(AIEventHandler.java:312) ~[?:?]
21.05 10:24:32 [Server] INFO at sun.reflect.GeneratedMethodAccessor6.invoke(Unknown Source) ~[?:?]
21.05 10:24:32 [Server] INFO at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_191]
21.05 10:24:32 [Server] INFO at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_191]
```

Love the plugin!

**Comment by RoboMWM** [comment]
Someday all will learn the usefulness of code blocks

**Comment by RoboMWM** [comment]
Hmm, this was probably fixed in #8. I guess I forgot to make a release for this plugin. Would be nice if github provided a way for me to view all open issues for all my projects in chronological order or something...
